### PR TITLE
Reorder installation guidelines in order of preference

### PR DESCRIPTION
**Pull request by samuel-w**
The pip guidelines should indicate in some way that it is not recommended.

Since official repos take time to update, should we put flatpak first?

### CODE REVIEW

**Comment by Hofer-Julian** [comment]
I also feel that flatpak should be first.
Then your PPA.
Then official distro commands
Then something like AUR

**Comment by samuel-w** [comment]
The distro commands vary by distro, so order doesn't matter. Looks best alphabetically

Issue is that flatpaks have poor system integration (lack of theming), and are more prone to sandboxing bugs.

**Comment by Hofer-Julian** [comment]
As far as I know flatpak does support theming, why would you think otherwise?

Sandboxing is a feature in my opinion, the bugs are quite minimal as far as I can tell. Just like your PPA it is an advantage that it is maintained by me some one who is also involved in Vorta. The problem with your PPA is that it there is no guarantee that it works on all debian based distros. I assume you only test a few Ubuntu versions.

**Comment by Hofer-Julian** [comment]
https://docs.flatpak.org/en/latest/desktop-integration.html#theming

**Comment by samuel-w** [comment]
It doesn't support all theming. I used to use XFCE with the gtk2 qt theme so that I can have the same qt and gtk theme, but flatpak left the theme as the default fusion.

**Comment by Hofer-Julian** [comment]
Maybe you have to install https://github.com/flathub/org.kde.PlatformTheme.QGnomePlatform. But not sure.

**Comment by samuel-w** [comment]
Its a general flatpak issue https://github.com/flatpak/flatpak/issues/2030.

Also there are issues with dependencies, since each flatpak bundles its own. 50MB for flatpak vs 1MB native. Not much but still. The dependencies don't get automatically updated with the distro (good since distro dependencies can be out of date, bad because bundled dependencies can be out of date).

For Vorta, sandboxing is minimal since repos need to be in any location, meaning something like writing to ~/.bashrc is simple.

If Flatpak, the universal solution, is recommended over the distro solution, what is the point of the distro packages then?

**Comment by Hofer-Julian** [comment]
Your points are very valid. There is no free lunch unfortunately. 

Regarding distro packages. It is not in our hand, people will create them anyways. But they are useful to mention because 
- it keeps people at least away from pip
- debian e.g. currently supports more architectures than flathub
- people might feel that the distro build has more eyes looking at the finished product

**Comment by samuel-w** [comment]
Changed it so flatpak is first, noted pypi as last option and recommended use of pipx instead of pip

**Comment by m3nu** [comment]
Not pipx, please. Those alternative tools come and go.

**Comment by samuel-w** [comment]
Ok, changed it.

**Comment by samuel-w** [comment]
Ok, changed it.

File: install/linux.md [hunk 1]
```diff
@@ -57,6 +47,15 @@ $ sudo apt-get install vorta
 Maintained by [@samuel-w](https://github.com/samuel-w).
 
 
+## Install from Source
+If the earlier options do not work, Vorta can be installed as a Python package using [PIP](https://pip.readthedocs.io/en/stable/installing/). First [install](https://borgbackup.readthedocs.io/en/stable/installation.html) Borg using the package of your distribution or via PyPI. The latter needs some additional [source packages](https://borgbackup.readthedocs.io/en/stable/installation.html#dependencies). Then install Vorta from PyPI. Your local Python version must be >= 3.6.
+```
+$ pipx install vorta
```
Comment (m3nu) [inline install/linux.md:53]: Missed one here

**Comment by samuel-w** [comment]
Instead of manually listing every single install method, maybe link to repology https://repology.org/project/vorta/versions?

**Pull request closed without merging**